src/staticanalysis/frontend/src/store.js

```diff
@@ -8,6 +8,8 @@ const PREFERENCES_KEY = 'mozilla-sa-dashboard'
 const TASKCLUSTER_INDEX = 'https://index.taskcluster.net/v1'
 const TASKCLUSTER_QUEUE = 'https://queue.taskcluster.net/v1'
 const TASKS_SLICE = 10
+const FINAL_STATES = ['done', 'error']
+const MAX_TTL = 2 * 3600 * 1000 // 2 hours in ms
```

Comment:
Can you add a comment here and where we set the deadline for the task so that we remember to keep them in sync?